CodeEdit/Features/UtilityArea/Views/UtilityAreaView.swift

```diff
@@ -43,6 +51,13 @@ struct UtilityAreaView: View {
                     .buttonStyle(.icon(isActive: model.isMaximized, size: 24))
                 }
             }
+            .colorScheme(
+                model.selectedTerminals.isEmpty
+                ? colorScheme
+                : matchAppearance && darkAppearance
+                ? themeModel.selectedDarkTheme?.appearance == .dark ? .dark : .light
+                : themeModel.selectedTheme?.appearance == .dark ? .dark : .light
+            )
```

Comment:
```suggestion
          .colorScheme(
              model.selectedTerminals.isEmpty ? 
                  colorScheme :
                  (matchAppearance && darkAppearance ?
                      (themeModel.selectedDarkTheme?.appearance == .dark ? .dark : .light) :
                      (themeModel.selectedTheme?.appearance == .dark ? .dark : .light)
                  )
          )
```
I'd suggest "simplifying" those conditions.

Reply:
@activcoding that all depends if we want our ternary operands at  the beginning or the end of lines. Maybe we discuss this in our meetup today.